packages/react-mutation-mapper/src/util/LollipopPlotUtils.spec.ts

```diff
@@ -130,32 +141,244 @@ describe('LollipopPlotUtils', () => {
         });
     });
 
-    // TODO disabled for now due to an incompatibility with JEST
```

Comment:
Would it be okay to remove this test? It seems like jest may not have a complete DOM implementation.

Reply:
Let's keep it as is for now? One day it might be possible to re-enable it :)

Reply:
I've just restored the commented test